expr/seriesaggregators.go

```diff
@@ -81,29 +86,13 @@ func crossSeriesAvg(in []models.Series, out *[]schema.Point) {
 // crossSeriesAvg would compute the new datapoint as 4/4=1 while
 // crossSeriesAvgZero would compute it as 4/5=0.8
 func crossSeriesAvgZero(in []models.Series, out *[]schema.Point) {
+	crossSeriesSum(in, out)
 	for i := 0; i < len(in[0].Datapoints); i++ {
```

Comment:
Seems like it would make more sense to loop over `out` in this scenario
```suggestion
	for i in range out {
```